### ISSUE TITLE
participantConditionSupported Controlled Vocabulary

### ISSUE THREAD

**Issue by thill-odi** [openactive/modelling-opportunity-data#241]
Several different classifications and taxonomies exist relevant to accessibility concerns, and in particular to the classification of impairments and afflictions requiring accessibility adaptation.

Within the OpenActive space, there are currently two vocabularies defined:

* the top-level vocabulary defined in relation to `accessibility support` https://www.openactive.io/accessibility-support/
* the ad hoc list evolved at https://github.com/openactive/modelling-opportunity-data/issues/67

In addition, various schemata are found in research in the area, as evidenced by the categories used in the Activity Alliance's 2020 [Annual Disability and Activity Survey](http://www.activityalliance.org.uk/how-we-help/research/5563-activity-alliance-annual-disability-and-activity-survey):

- Long-term pain
- Mobility
- Chronic health condition
- Breathing / stamina
- Dexterity
- Mental health
- Hearing
- Memory
- Learning
- Social / behavioural
- Vision
- Speech

Considered internationally, a number of classification schemes exist, with the Washington Group Enhanced Short Set being widely used and being broadly comparable the UK government definition under the Equality Act (see further [Measuring Disability: Comparing Approaches](https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/disability/articles/measuringdisabilitycomparingapproaches/2019-08-06) ).

While the number of approaches and their lack of close alignment means harmonisation may be challenging, the benefits of a consistent SKOS (or perhaps SKOS-like) vocabulary for indicating conditions supported are clear: greater findability and filterability for end-users, carers, and link workers needing a clear sense of activity suitability.

Developing such a taxonomy, specifically adapted to physical-activity use cases, will need to be a collaborative endeavour across numerous authorities and interest groups. However, three starting points seem clear:

* a hierarchical model, with broad top terms dividing into narrower and more specialised terms, seem preferable to a flat list
* this hierarchy can be arbitrarily deep: while broad terms may be sufficient for most e.g. leisure centres, social-prescribing and therapeutic use-cases may demand more precision
* a governance structure similar to that of the existing OpenActive Activity Committee should form a useful starting point for defining and guiding the evolution of this list.

**Comment by nickevansuk** [comment]
This looks great! Two quick comments:

If this is to be used similar to the activity list, all prefLabels should be self-describing without the context of the hierarchy - as they may be displayed on their own. So e.g “Support Required” should be “Mobility Support Required”

Perhaps the prefLabels should be the most widely understood by the intended audience, with the altLabels used for technical terms (is these are not as widely understood). Hence prioritising the list user over the maintainer of the list.

**Comment by thill-odi** [comment]
Yes, a +1 to both of those. I think technical terms will be valuable in the accessibility space, so should be retained - but as you say, as altLabels rather than prefLabels. This in turn may lead to difficulties where applied and expert vocabularies don't quite cleave at the same point - but, then, resolving that is the job of the notional curation committee.

**Comment by thill-odi** [comment]
Note that the Activity Alliance have a controlled vocabulary of fourteen terms they user for describing health conditions. Locate and compare.

**Comment by thill-odi** [comment]
Revised schema incorporating points raised by Activity Alliance:

* Learning Difficulties
* Mental Health Needs
* Neurological Impairment
    Brain Injury
    Cerebral Palsy
    Dementia
    Epilepsy
    Multiple Sclerosis
    Parkinson's Disease
    Stroke
* Physical Impairment
    Arthritis
    Breathing Impairment
    Dexterity Impairment
    Long-term Pain
    Mobility Impairment
        Amputation
            Lower Limb Amputee
            Upper Limb Amputee
        Restricted Growth
            Dwarfism
        Support Required
        Upper Limb Amputee
        Wheelchair Required
    Osteoporosis
    Overweight/Obesity
    Recuperative
        Back Injury
        Cancer Support
            Breast Cancer Support
        Cardiac
        Post-surgical
    Stamina Impairment
* Pregnancy Support
    Antenatal
    Postnatal
* Profound and Multiple Learning Disabilities (PMLD)
* Sensory Impairment
    Visual Impairment
    Hearing Impairment
* Social/Behavioural Needs
    Autism-spectrum
    Communication Needs
        Speech Impairment
    Sensory Processing Disorder

**Comment by thill-odi** [comment]
Additional notes/issues regarding the above:

- Arthritis is often associated in therapeutic terms with osteoporosis
- Sensory Processing Disorder is often reported by those diagnosed with it as a sensory impairment
